vowpalwabbit/cb_adf.cc

```diff
@@ -29,37 +29,53 @@ namespace CB_ADF
 {
 struct cb_adf
 {
-  vw* all;
-
+ public:
   cb_to_cs_adf gen_cs;
```

Comment:
Can this be made private?  
1) Looks like cb_type needs to be set after construction.   This can be done with a get/set in cb_adf.  
2) Looks like scorer needs to be set.  But this can be passed as an arg to cb_adf constructor